k8s/locals.tf

```diff
@@ -50,7 +50,12 @@ locals {
 
   external_lb_certificate = {
     filecoin-dev-apn1-glif-eks = [
+      "*.${var.route53_domain}"
+    ]
+    filecoin-mainnet-apn1-glif-eks = [
       "*.${var.route53_domain}",
+      "filecoin.tools",
+      "*.filecoin.tools",
       "calibration.node.glif.io",
       "*.calibration.node.glif.io",
       "calibration.filecoin.tools",
```

Comment:
needs to be careful with applying these changes, because it can affect load balancing.